PRODUCTIONISATION.md

```diff
@@ -19,26 +19,30 @@
 
 Ensuring that your project adheres to the following RFCs is a good first step as these will be referenced further below:
 
+* [RFC 008 - Server naming convention][rfc_008]
 * [RFC 011 - Editorconfig][rfc_011]
 * [RFC 012 - Makefile standards][rfc_012]
 * [RFC 013 - Code repository standards][rfc_013]
 
 ### 1. General requirements
 
 * [ ] 1.1 Application source code hosted with a Git VCS service
-* 1.2 No linting violations
-  * [ ] 1.2.1 No style related linting violations and no overrides permitted
-  * [ ] 1.2.2 No complexity related linting violations. Overrides permitted but must be commented inline with the code it is overriding
+* 1.2 Adhere to project coding standards
+  * [ ] 1.2.1 Provides a style guideline
+  * [ ] 1.2.2 Consistency within development team
+  * [ ] 1.2.3 If applicable, no linting violations (style and complexity related)
```

Comment:
generalised wording would be "enforce coding standards", which is then always applicable
```suggestion
  * [ ] 1.2.3 Enforce coding standards (e.g. automated linter on code changes)
```